src/auth/auth.service.ts

```diff
@@ -70,9 +66,12 @@ export class AuthService {
         `User '${user.username}' has already 200 tokens and can't have anymore`,
       );
     }
-    const secret = bufferToBase64Url(randomBytes(54));
+    const secret = bufferToBase64Url(randomBytes(64));
     const keyId = bufferToBase64Url(randomBytes(8));
-    const accessToken = await hashPassword(secret);
+    const accessToken = crypto
```

Comment:
I think that it should be named `accessTokenHash` to make clearer it is not the actual token.